keeper-gradle-plugin/build.gradle.kts

```diff
@@ -78,11 +80,20 @@ val agpVersion = findProperty("keeperTest.agpVersion")?.toString() ?: defaultAgp
 
 // See https://github.com/slackhq/keeper/pull/11#issuecomment-579544375 for context
 val releaseMode = hasProperty("keeper.releaseMode")
+val shade: Configuration = configurations.maybeCreate("compileShaded")
+configurations.getByName("compileOnly").extendsFrom(shade)
```

Comment:
I'd expect to see "implementation" here instead of "compileOnly", because I believe we want the shaded version of zipflinger to be on the runtime classpath and not just the compile classpath. (I could certainly be wrong; I haven't used this shadow plugin much.) I also see that the plugin creates a `shadow` configuration itself and wires that up as the configuration for the `shadowJar` task, could we use that here?

Reply:
We use compileOnly to make it available at compile time but the shade plugin relocates everything afterwards, so we wouldn't want it on implementation and pulling in its dependencies

Reply:
I think I follow... 👍